src/translations.js

```diff
@@ -473,6 +473,40 @@ export const Translations = [
       select: 'Sprache auswählen',
     },
   },
+  {
+    id: 'de-CH',
+    name: 'Schweizer Deutsch',
```

Comment:
```suggestion
    name: 'Deutsch - Schwizerdütsch',
```
I suggest changing it to "Deutsch Schweiz" instead so that it appears next to "Deutsch - Hochdeutsch", just like with the Portugese translations.